hub-saml/src/main/java/uk/gov/ida/saml/hub/transformers/outbound/EidasUnsignedAssertionsTransformer.java

```diff
@@ -28,21 +30,31 @@
 
     private final OpenSamlXmlObjectFactory openSamlXmlObjectFactory;
     private final AuthnContextFactory authnContextFactory;
+    private final String hubEidasEntityId;
 
-    public EidasUnsignedAssertionsTransformer(OpenSamlXmlObjectFactory openSamlXmlObjectFactory, AuthnContextFactory authnContextFactory) {
+    public EidasUnsignedAssertionsTransformer(OpenSamlXmlObjectFactory openSamlXmlObjectFactory, AuthnContextFactory authnContextFactory, String hubEidasEntityId) {
         this.openSamlXmlObjectFactory = openSamlXmlObjectFactory;
         this.authnContextFactory = authnContextFactory;
+        this.hubEidasEntityId = hubEidasEntityId;
     }
 
     public Assertion transform(HubEidasAttributeQueryRequest originalQuery) {
 
         CountrySignedResponseContainer countrySignedResponseContainer = originalQuery.getCountrySignedResponseContainer().get();
         Assertion assertion = openSamlXmlObjectFactory.createAssertion();
-        assertion.setIssueInstant(DateTime.now());
+        DateTime now = DateTime.now();
+        assertion.setIssueInstant(now);
         String unsignedCountryIssuer = countrySignedResponseContainer.getCountryEntityId();
         assertion.setIssuer(openSamlXmlObjectFactory.createIssuer(unsignedCountryIssuer));
         assertion.setID(UUID.randomUUID().toString());
 
+        Conditions conditions = openSamlXmlObjectFactory.createConditions();
+        conditions.setNotBefore(now);
+        conditions.setNotOnOrAfter(now.plusMinutes(5));
```

Comment:
This expiry is based on an example of five minutes set in the [eIDAS message format spec](https://ec.europa.eu/cefdigital/wiki/download/attachments/82773108/eidas_message_format_v1.0.pdf?version=1&modificationDate=1497252920416&api=v2), and as seen in Stub IDP